README.md

```diff
@@ -297,10 +354,73 @@ return (
 );
 ```
 
-### `useCombinedAPIQueries`
+### `useSuspenseCombinedAPIQueries`
 
 A helper for combining multiple parallel queries into a single `react-query`-like hook.
 
+Queries performed using this hook are cached independently, just as if they had been performed individually using `useSuspenseAPIQuery`.
+
+```typescript
+const query = useSuspenseCombinedAPIQueries(
+  ['GET /messages', { filter: 'some-filter' }],
+  ['GET /messages/:id', { id: 'some-message-id' }],
+);
+
+// Here all queries are complete - pending and error states are handled by suspense and error boundaries
+
+query.data; // [Message[], Message]
+
+const [list, message] = query.data;
+
+list; // Message[]
+message; // Message
+```
+
+#### `isFetching`
+
+Indicates whether _at least one_ query is in the "fetching" state.
+
+#### `isRefetching`
+
+Indicates whether _at least one_ query is in the "refetching" state.
+
+#### `refetchAll()`
```

Comment:
Is it worth re-documenting all of these props if they're exactly the same as the ones in `react-query`?

Reply:
for sure - could be worth trying to improve it here. I think for now I'd like to just have something then maybe I can follow up. react-query docs take an approach where its documented on the non-suspense hook and then the suspense version points to that with some exclusions

Reply:
Sorry, I might not have phrased this correctly to get my point across. I was actually saying it to mean that I don't think it's worth redocumenting all the react-query props, since we could just link to their docs and say we expose the exact same API.